connect/src/spirc.rs

```diff
@@ -817,17 +820,24 @@ impl SpircTask {
             // Check for malformed gid
             let tracks_len = self.state.get_track().len() as u32;
             let mut track_ref = &self.state.get_track()[index as usize];
-            while track_ref.get_gid().len() != 16 {
-                warn!(
-                    "Skipping track {:?} at position [{}] of {}",
-                    track_ref.get_uri(),
-                    index,
-                    tracks_len
-                );
-                index = if index + 1 < tracks_len { index + 1 } else { 0 };
-                track_ref = &self.state.get_track()[index as usize];
-            }
-            SpotifyId::from_raw(track_ref.get_gid()).unwrap()
+
+            if track_ref.get_uri().starts_with("spotify:episode:")
```

Comment:
This can also be `spotify:show:xxx`

Reply:
Are you sure?
In this context, we are loading a specific track (or, in this case, episode) to play it and not only trying to get meta information about a piece of audio(collection)

Reply:
Quite sure, add a [debug of the `context_uri`](https://github.com/ashthespy/librespot/commit/21ca960ccd8184cdc1cc55a5a0a9e64a8c320b81#diff-fb4606287b6e129f05bf85e440522421R696), then go to `Browse>Top Podcasts>` and hit play on any of the shows, you'll see a `spotify:show:xxx` URI..